packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChartRTL.test.tsx

```diff
@@ -1,13 +1,551 @@
+    },
+  );
+
+  testWithWait(
+    'Should show the single callout over the bar on mouse over',
+    GroupedVerticalBarChart,
+    { data: chartPoints, isCalloutForStack: false, calloutProps: { doNotLayer: true } },
+    container => {
+      // Arrange
+      const bars = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'rect');
+      expect(bars).toHaveLength(6);
+      fireEvent.mouseOver(bars[0]);
+      // Assert
+      expect(getByClass(container, /calloutlegendText/i)).toBeDefined();
+      expect(getByClass(container, /calloutlegendText/i)).toHaveLength(1);
+    },
+  );
+
+  testWithWait(
+    'Should show the custom callout over the bar on mouse over',
+    GroupedVerticalBarChart,
+    {
+      data: chartPoints,
+      calloutProps: { doNotLayer: true },
+      onRenderCalloutPerDataPoint: (props: IVSChartDataPoint) =>
+        props ? (
+          <div className="onRenderCalloutPerDataPoint">
+            <p>Custom Callout Content</p>
+          </div>
+        ) : null,
+    },
+    container => {
+      const bars = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'rect');
+      fireEvent.mouseOver(bars[0]);
+      // Assert
+      expect(getById(container, /toolTipcallout/i)).toBeDefined();
+      expect(screen.queryByText('Custom Callout Content')).toBeDefined();
+    },
+  );
+});
+
+describe('Grouped vertical bar chart - Subcomponent Labels', () => {
+  testWithWait(
+    'Should render the xAxis label based on noOfCharsToTruncate',
+    GroupedVerticalBarChart,
+    { data: chartPoints, showXAxisLablesTooltip: true, noOfCharsToTruncate: 3 },
+    container => {
+      // Assert
+      expect(getById(container, /showDots/i)).toHaveLength(3);
+      expect(getById(container, /showDots/i)[0]!.textContent!).toEqual('Met...');
+    },
+  );
+
+  testWithWait(
+    'Should show rotated x-axis labels',
+    GroupedVerticalBarChart,
+    { data: chartPoints, rotateXAxisLables: true },
+    container => {
+      // Assert
+      expect(getByClass(container, /tick/i)[0].getAttribute('transform')).toContain('rotate(-45)');
+    },
+  );
+
+  testWithWait(
+    'Should show the x-axis labels tooltip when hovered',
+    GroupedVerticalBarChart,
+    { data: chartPoints, showXAxisLablesTooltip: true },
+    container => {
+      const bars = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'rect');
+      expect(bars).toHaveLength(6);
+      fireEvent.mouseOver(bars[0]);
+      // Assert
+      expect(getById(container, /showDots/i)).toHaveLength(3);
+      expect(getById(container, /showDots/i)[0]!.textContent!).toEqual('Meta...');
+    },
+  );
+
+  testWithWait(
+    'Should render the xAxis and yAxis with specified tick count',
+    GroupedVerticalBarChart,
+    { data: chartPoints, yAxisTickCount: 5 },
+    container => {
+      // Assert
+      expect(getByClass(container, /tick/i)).toHaveLength(9);
```

Comment:
Does 9 equal x tick count + y tick count

Reply:
Yes

Reply:
Can you mention it as a comment in the code.